core/capabilities/ccip/common/common.go

```diff
@@ -8,6 +8,11 @@ import (
 	"github.com/smartcontractkit/chainlink-integrations/evm/utils"
 )
 
+type ExtraDataDecoded struct {
+	ExtraArgsDecoded    map[string]any
+	DestExecDataDecoded []map[string]any
+}
```

Comment:
Can we have doc comments on public methods, members, and types that are introduced?

Reply:
```suggestion
// ExtraDataDecoded contains a generic representation of chain specific message parameters. A
// map from string to any is used to account for different parameters required for sending messages
// to different destinations.
type ExtraDataDecoded struct {
	// ExtraArgsDecoded contain message specific extra args.
	ExtraArgsDecoded    map[string]any
	// DestExecDataDecoded contain token transfer specific extra args.
	DestExecDataDecoded []map[string]any
}
```